go/vt/vtgate/executor_framework_test.go

```diff
@@ -177,7 +177,10 @@ var executorVSchema = `
 			"params": {
 				"region_bytes": "1"
 			}
-    	}
+    	},
+		"cfc": {
+          "type": "cfc"
```

Comment:
The formatting looks off. I'm surprised that `gofmt` is passing though.

Reply:
This is in a string, so I think that's why `gofmt` is not complaining

Reply:
fixed.